compliance/collection/main.go

```diff
@@ -283,7 +407,7 @@ func main() {
 	if features.RHCOSNodeScanning.Enabled() {
 		rescanInterval := env.NodeRescanInterval.DurationSetting()
 		cmetrics.ObserveRescanInterval(rescanInterval, getNode())
-		log.Infof("Node Rescan interval: %s", rescanInterval.String())
+		log.Infof("Node Rescan interval: %s - caching results for %s", rescanInterval.String(), env.NodeInventoryCacheDuration.DurationSetting())
```

Comment:
[This isn't actually a nitpick when it's about a code smell.] If the logging message has changed but the surrounding code has **not** changed that's the symptom that logging change must go in a different location, that's near to where relevant things actually happen.

Specifically here, the message `Node Rescan interval: %s` informed the user about the value of the corresponding setting.
Newly added `caching results for %s` tells that something is **happening** whereas it's not happening here.

Reply:
The idea was to tell the user what durations are configured - for the rescan as well as the cache.
Which leads to the conclusion that the wording may not be optimal.
I have updated the message text with better wording. Please TAL.

Reply:
The message is better, but the location isn't very inspiring. Let this sit with me and we keep it as-is unless it provokes me again before the end of the work on this PR.